src/BaseStyles.tsx

```diff
@@ -34,7 +32,10 @@ export type BaseStylesProps = ComponentProps<typeof Base>
 
 function BaseStyles(props: BaseStylesProps) {
   const {children, ...rest} = props
-  useMouseIntent()
+
+  // load polyfill for :focus-visible
+  require('focus-visible')
```

Comment:
Does this need to be loaded in the body of this function?

Reply:
I did this to follow the pattern from `useMouseIntent`.  If `BaseStyles` was never used, `useMouseIntent` never ran.  Figured the same logic should apply to the `focus-visible` class being applied with the polyfill as well.  I'm happy to move it to the top imports instead if you think we are safe to always pull in the polyfill.

Reply:
Just a sanity check: this is a no-op after the first execution, right?

Reply:
Yep, definitely only runs the required code once.

Reply:
This mixes `require` into modules causing previously working (with v27) rollup builds to fail.

Reply:
@daaku thanks for pointing this out!  #1207 opened to track this